### PR TITLE
Updating references to bitbucket

### PR DESCRIPTION
**Pull request by jasonamyers**
Fixes: python-virtualenvwrapper/virtualenvwrapper#9

### CODE REVIEW

**Comment by dhellmann** [comment]
This looks good. I'm still working on getting the CI jobs to actually pass (#21 ) and then setting up Mergify.com to enforce the jobs running and passing on PRs (#5 ). I'd like to use this PR to test some of those changes.

**Comment by dhellmann** [comment]
I believe if you merge #24 then rebase this PR I can approve it and Mergify will merge it.

**Comment by dhellmann** [comment]
@Mergifyio rebase

File: docs/source/extensions.rst [hunk 1]
```diff
@@ -45,13 +45,13 @@ Templates
 Below is a list of some of the templates available for use with
 :ref:`command-mkproject`.
 
-.. _templates-bitbucket:
+.. _templates-github:
```
Comment (dhellmann) [inline docs/source/extensions.rst:48]: I think we don't want this change, the template is still bitbucket.

File: docs/source/projects.rst [hunk 2]
```diff
@@ -23,13 +23,13 @@ A new project directory can be created empty, or populated using one
...
-repository from a project on bitbucket and create a new Django
-site, combine the :ref:`templates-bitbucket` and
+repository from a project on GitHub and create a new Django
+site, combine the :ref:`templates-github` and
 :ref:`templates-django` templates.
 
 ::
 
-    $ mkproject -t bitbucket -t django my_site
+    $ mkproject -t github -t django my_site
```
Comment (dhellmann) [inline docs/source/projects.rst:32]: Same here, the example template is for a bitbucket project, not GitHub, so we don't want the changes to this file.

**Comment by jasonamyers** [comment]
@dhellmann are we ready to merge this?

File: docs/source/projects.rst (hunk 2 above)
Comment (dhellmann) [reply docs/source/projects.rst:32]: Somehow this wasn't reverted?

Comment (jasonamyers) [reply docs/source/projects.rst]: how does one save files... fixed it

**Review by dhellmann — APPROVED** [review]
This looks great now. Let's see if the bot is set up properly.

**Pull request merged**